database/emotes.sql

```diff
@@ -24,6 +30,7 @@ CREATE INDEX "emote_files_file_id_index" ON "emote_files" ("file_id");
 CREATE TABLE "emote_sets" (
     "id" uuid PRIMARY KEY,
     "owner_id" uuid, -- Ref: users.id -> On Delete Cascade
+    "hidden_on_profile" bool NOT NULL DEFAULT FALSE,
```

Comment:
perhaps this could be a flag 🤔

Reply:
not sure, I thought flags are meant to be public flags not related to the owner

Reply:
they can be anything